src/components/Header.jsx

```diff
@@ -123,15 +107,22 @@ const languages = {
 };
 
 function Header() {
-  const classes = useStyles();
   const { t, i18n } = useTranslation();
   const [anchorEl, setAnchorEl] = useState(null);
 
   return (
     <Container>
       <UnderConstructionAlert t={t} />
       <OnRenderSnackbar />
-      <Grid container className={classes.header} spacing={3}>
+      <Grid
+        container
+        sx={{
+          flexGrow: 1,
+          padding: 3,
```

Comment:
`p:3`

Reply:
I'm going to keep this one as there's lots of styling going on here and the shorthand doesn't help condense this into one line :P